cmd/service/main.go

```diff
@@ -114,7 +114,7 @@ func run() error {
 func createNode(ctx context.Context, cfg *config.Config, db database.DB) (universe.Node, error) {
 	worlds := worlds.NewWorlds(db)
 	assets2d := assets_2d.NewAssets2d(db)
-	assets3d := assets_3d.NewAssets3d(db)
+	assets3d := assets_3d.NewAssets3d(cfg, db)
```

Comment:
please do it in different way
we have the config in our main context
look https://github.com/momentum-xyz/ubercontroller/blob/develop/universe/worlds/worlds.go#L44-L47